### PR TITLE
python3-jsonschema: update to 4.9.0.

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/python3-jsonschema/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ license="MIT"
 homepage="https://github.com/Julian/jsonschema"
 changelog="https://raw.githubusercontent.com/Julian/jsonschema/master/CHANGELOG.rst"
 distfiles="${PYPI_SITE}/j/jsonschema/jsonschema-${version}.tar.gz"
-checksum=ec2802e6a37517f09d47d9ba107947589ae1d25ff557b925d83a321fc2aa5d3b
+checksum=73764f461d61eb97a057c929368610a134d1d1fffd858acfe88864ee94f1f1d3
 # Tests require jsonschema to be installed already. Setting PYTHONPATH doesn't seem to do trick
 make_check=no
```
Comment (paper42) [inline srcpkgs/python3-jsonschema/template:17]: But I am not sure if it's worth the setup. Is this common in pep517 packages? If yes, could we add something like this to the build style?
```suggestion

do_check() {
	rm -rf *.dist-info tmp # remove artifacts from previous unsuccessful check run
	pip install --no-deps --target tmp ./jsonschema-$version-py3-none-any.whl
	mv tmp/*.dist-info .
	PYTHONPATH=. pytest
	rm -rf *.dist-info tmp
}
```

Comment (jcgruenhage) [reply srcpkgs/python3-jsonschema/template:17]: I've seen it a few times, not sure if it's enough to call it common but I've definitely seen it a few times. Other distros handle this in a similar way, so if we could include it in the build style that'd be great, although I don't feel confident enough in quite a bit of this python stuff, so I'd appreciate it if someone else could look into the check/build-style stuff.

Comment (jcgruenhage) [reply srcpkgs/python3-jsonschema/template:17]: Aside of that, in this specific case: I ran the test cases of ansible-lint, which depends on this, and I've run ansible-lint on my local repositories, to test that this release works.

Comment (jcgruenhage) [reply srcpkgs/python3-jsonschema/template:17]: Aside of that, in this specific case: I ran the test cases of ansible-lint, which depends on this, and I've run ansible-lint on my local repositories, to test that this release works.

Comment (paper42) [reply srcpkgs/python3-jsonschema/template:17]: Let's add this for now, we can easily remove it if/when we figure out how to do this (if we can and want to) universally.

Comment (jcgruenhage) [reply srcpkgs/python3-jsonschema/template:17]: Okay, so I'm back on this, and as it is right now it doesn't work. With a few more changes the tests now do run successfully, and I'm currently running the tests for everything depending on python3-jsonschema. ooi: is there a nice way to do this with xtools or xbps-src? I'm looping over all packages returned by `grep python3-jsonschema -r -l srcpkgs | cut -d/ -f2` right now, but that seems like a common enough usecase for checking whether something breaks when updating a library package.

Comment (paper42) [reply srcpkgs/python3-jsonschema/template]: I don't think there is a tool for that

**Pull request merged**